latest/index.bs

```diff
@@ -175,7 +175,264 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+│
+└── 123.zarr
+    └── table                 # The table group is a container which holds a table that is compatible with AnnData.
+        │                     # The table group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image, labels, or points group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "@type", which is set to `"ngff:region_table"`
+        |                     # `.zattrs` MUST contain "region", which is the path to the data the table is annotating.
+        |                     # "region" MUST be a string (single region) or an array of strings (multiple regions).
+        |                     # `.zattrs` MUST contain "region_key" if "region" is an array. "region_key" is the key in `obs` denoting which region a given row corresponds to.
+        |                     # `.zattrs` MAY contain "instance_key", which is the key in `obs` that denotes which instance in "region" the row corresponds to. If "instance_key" is not provided, the values from the `obs` `.zattrs` "_index" key is used.
+        │
+        ├── X                 # You MAY add an zarr array `X`.
+        │   │                 # `X` MUST not be a complex type (i.e., MUST be a single type)
+        │   │                 # `X` MAY be chunked as the user desires.
+        │   ├── .zarray
+        │   ├── 0.0
+        │   │   ...
+        │   └── n.m
+        |
+        ├── layers            # You MAY add a `layers` group, which contains dense matrices with the same shape as X.
+        │   │
+        │   ├── .zgroup
+        │   ├── .zattrs
+        │   │
+        │   └── layer_0       # You MAY add a zarr array for each layer
+        |       |             # Each layer array MUST have the same shape as X
+        |       |             # Each layer array SHOULD be chunked the same as X
+        |       ├── .zarray
+        |       |
+        |       ├── 0.0
+        │       │   ...
+        │       └── n.m
+        │        
+        ├── obs               # You MUST add an obs group container. The obs group holds a table of annotations on the rows in X.
+        │   │                 # The rows in obs MUST be index-matched to the rows in X.
+        │   ├── .zgroup
+        │   │                     
+        │   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+        │   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+        │   │                 # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+        │   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        │   │      
+        │   └── col_0         # Each column in the obs table is a 1D zarr array. The rows can be chunked as the user desires.
```

Comment:
This isn't quite true. For example, we encode [nullable integers](https://anndata.readthedocs.io/en/latest/fileformat-prose.html#nullable-integers-and-booleans) and [categories](https://anndata.readthedocs.io/en/latest/fileformat-prose.html#categorical-arrays) as groups.